--- Makefile	2012-01-16 17:03:47.000000000 +0900
+++ Makefile	2012-01-18 01:23:54.000000000 +0900
@@ -37,6 +37,7 @@
 	@echo "  changes    to make an overview of all changed/added/deprecated items"
 	@echo "  linkcheck  to check all external links for integrity"
 	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
+	@echo "  pdf        to to make pdf file"
 
 clean:
 	-rm -rf $(BUILDDIR)/*
@@ -151,3 +152,8 @@
 	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
+
+pdf:
+	$(SPHINXBUILD) -b pdf $(ALLSPHINXOPTS) _build/pdf
+	@echo
+	@echo "Build finished. The PDF files are in _build/pdf."
--- source/conf.py	2012-01-16 17:03:47.000000000 +0900
+++ source/conf.py	2012-01-18 01:22:29.000000000 +0900
@@ -240,3 +240,14 @@
 
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
+
+# pdf
+extensions += ['rst2pdf.pdfbuilder']
+pdf_documents = [ 
+    (master_doc, project, project, copyright),
+]
+pdf_stylesheets = ['sphinx','kerning','a4','ja']
+pdf_language = "ja"
+pdf_use_coverpage = True
+pdf_use_toc = True
+
--- ja.json	2012-01-18 01:30:58.000000000 +0900
+++ ja.json	2012-01-18 01:29:07.000000000 +0900
@@ -0,0 +1,13 @@
+{
+  "fontsAlias" : {
+    "stdFont": "HeiseiKakuGo-W5",
+    "stdBold": "HeiseiKakuGo-W5",
+    "stdItalic": "HeiseiKakuGo-W5",
+    "stdBoldItalic": "HeiseiKakuGo-W5",
+    "stdMono": "Ricty"
+  },
+  "styles" : [
+        ["base",    { "wordWrap": "CJK"}],
+        ["literal", { "wordWrap": "None"}],
+        ["normal",  { "wordWrap": "CJK"}]]
+}
--- source/_templates/searchbox.html    2012-01-18 01:44:33.000000000 +0900
+++ source/_templates/searchbox.html    2012-01-18 01:44:39.000000000 +0900
@@ -0,0 +1 @@
+ 
